listenbrainz/webserver/templates/index/index.html

```diff
@@ -74,6 +74,19 @@ <h2 class="page-title">Welcome to ListenBrainz!</h2>
             {% endif %}
             </tbody>
           </table>
+          <table class="table table-collapsed">
+          <thead>
+            <tr>
+              <th>Latest Listens</th>
+            </tr>
+          </thead>
+          <tbody>
+          {% if latest_listens %}
```

Comment:
this `if` block needs a corresponding `endblock`. See the [Jinja2 documentation](http://jinja.pocoo.org/docs/2.10/) for syntax.